Cargo.toml

```diff
@@ -1,16 +1,10 @@
 [workspace]
 resolver = "2"
-members = [
-    "api",
-    "pg_replicate",
-    "postgres",
-    "replicator",
-    "telemetry"
-]
+members = ["api", "supabase_etl", "postgres", "replicator", "telemetry"]
 
 [workspace.dependencies]
 api = { path = "api" }
-pg_replicate = { path = "pg_replicate" }
+supabase_etl = { path = "supabase_etl" }
```

Comment:
Initially I thought that maybe having the crate named core would have been better and less confusing, especially once we start splitting in more crates. What do you think?

Reply:
We can rename crates to whatever makes sense in a new PR especially since none of the crates are published on crates.io yet.